pxr/imaging/plugin/hdRpr/rifcpp/rifContext.cpp

```diff
@@ -268,6 +268,7 @@ ContextMetal::ContextMetal(rpr::Context* rprContext, std::string const& modelPat
 }
 
 std::unique_ptr<Image> ContextMetal::CreateImage(HdRprApiFramebuffer* rprFrameBuffer) {
+#ifdef __APPLE__
```

Comment:
They reverted new behavior to the old one - https://github.com/Radeon-Pro/RadeonProImageProcessing/pull/392, so that `rifContextCreateImageFromMetalMemory` is not present in RadeonImageFilters lib.